root-feature/Navigation/TabNavigation.tsx

```diff
@@ -0,0 +1,39 @@
+import BottomNavTabBar from "@components/bottomTabComponents/BottomNavTabBar"
+import { createBottomTabNavigator } from "@react-navigation/bottom-tabs"
+import { getFocusedRouteNameFromRoute } from "@react-navigation/native"
+import { ProfileStack } from "@screens/ProfileScreen/Navigation/ProfileScreensNavigation"
+import React from "react"
+
+const Tab = createBottomTabNavigator()
+
+const getTabBarVisibility = (route: any) => {
+  const routeName = getFocusedRouteNameFromRoute(route)!
+  const tabHiddenRoutes = [
+    "EditProfileScreen",
+    "SettingsScreen",
+    "ChangePasswordScreen"
+  ]
+  if (tabHiddenRoutes.includes(routeName)) {
+    return false
+  }
+  return true
+}
```

Comment:
We don't need this, we'll put the screens that don't use the tab bar in the root stack navigator.